azure-pipelines.yml

```
@@ -9,7 +9,7 @@ resources:
   repositories:
     - repository: cnp-azuredevops-libraries
       type: github
-      ref: refs/heads/master
+      ref: refs/heads/pass-init-subscription-throug
```

https://github.com/hmcts/cnp-azuredevops-libraries/pull/94
```suggestion
      ref: refs/heads/master
```